pkg/metrics/metrics.go

```diff
@@ -40,12 +40,18 @@ func InitializeMetrics() {
 		utils.ForwardBytesGaugeName,
 		forwardBytesGaugeDescription,
 		utils.Direction)
-	WindowsGauge = exporter.CreatePrometheusGaugeVecForMetric(
+	HnsStatsGauge = exporter.CreatePrometheusGaugeVecForMetric(
```

Comment:
This should be `HNSStatsGauge`

https://go.dev/wiki/CodeReviewComments#initialisms

Reply:
Thank you @timraymond, I wasn't aware of the convention! Updated PR accordingly 😊